test/unit/Cardano/WalletLayerSpec.hs

```diff
@@ -0,0 +1,127 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Cardano.WalletLayerSpec
+    ( spec
+    ) where
+
+
+import Prelude
+
+import Cardano.DBLayer
+    ( DBLayer (..), PrimaryKey (..) )
+import Cardano.DBLayer.MVar
+    ( newDBLayer )
+import Cardano.NetworkLayer.HttpBridgeSpec
+    ( mockNetworkLayer, noLog )
+import Cardano.Wallet
+    ( WalletId (..), WalletName (..) )
+import Cardano.Wallet.AddressDerivationSpec
+    ()
+import Cardano.Wallet.AddressDiscovery
+    ( SeqState )
+import Cardano.Wallet.AddressDiscoverySpec
+    ()
+import Cardano.Wallet.MnemonicSpec
+    ()
```

Comment:
Same here. In practice, `Arbitrary` instances are local to a test because generators are tailored to the properties they test. I'd advise to re-define the instances here in this spec file, and goes for generic arbitrary as a default if it's enough to cover the need for this tests.

Reply:
now, no other module arbitrary imports